src/tools/clippy/clippy_lints/src/manual_retain.rs

```diff
@@ -12,12 +12,8 @@ use rustc_semver::RustcVersion;
 use rustc_session::{declare_tool_lint, impl_lint_pass};
 use rustc_span::symbol::sym;
 
-const ACCEPTABLE_METHODS: [&[&str]; 4] = [
-    &paths::HASHSET_ITER,
-    &paths::BTREESET_ITER,
-    &paths::SLICE_INTO,
-    &paths::VEC_DEQUE_ITER,
-];
+const ACCEPTABLE_METHODS: [&[&str]; 4] =
```

Comment:
Please don't push formatting changes to Clippy.

I wonder why this is happening recently in every other PR :thinking: